InternalError("Throwable(Object(Some(Object(RwLock { data: Object(java/lang/InternalError)
  backtrace=java/lang/StackTraceElement[5]
  detailMessage=String(\"Operand stack overflow\")
  cause=Object(class java/lang/InternalError)
  stackTrace=java/lang/StackTraceElement[0]
  depth=int(5)
  suppressedExceptions=Object(class java/util/Collections$EmptyList)
 })))):
stdout: === Module Annotations and Metadata Tests ===
--- Test: Module Annotations ---
